src/utils/api/helpers/ecommerce/ecommercePaymentMethodsHelper.ts

```diff
@@ -247,6 +250,7 @@ export const getPaymentInstruments = async (
                   onError(ErrorsType.UNAUTHORIZED);
                   return [];
                 default:
+                  onError(ErrorsType.EMPTY_METHOD);
```

Comment:
why adding this error type here? I would use GENERIC_ERROR here too (since there is no need to specialize error here). moreover the default case here is not assimilable to an empty method list returned by b.e.